src/templates/tr/base.html

```diff
@@ -0,0 +1,184 @@
+
+{% block home %}Ana Sayfa{% endblock %}
+{% block table_of_contents_title %}İçindekiler Tablosu{% endblock %}
+{% block contributors_title %}Katkıda Bulunanlar{% endblock %}
+{% block methodology_title %}Metodoloji{% endblock %}
+
+{% block part %}Kısım{% endblock %}
+{% block chapter %}Bölüm{% endblock %}
+
+{% block start_exploring %}Keşfetmeye başlayın{% endblock %}
+
+{% block menu_title %}Sayfa menüsü{% endblock %}
+{% block open_the_menu %}Menüyü açın{% endblock %}
+{% block close_the_menu %}Menüyü kapatın{% endblock %}
+
+{% block index_nav_title %}İçindekiler tablosu bölümü{% endblock %}
+{% block index_title %}İndeks{% endblock %}
+{% block open_the_index %}İçindekiler Tablosunu Açın{% endblock %}
+{% block close_the_index %}İçindekiler Tablosunu Kapatın{% endblock %}
+
+{% block and %}ve{% endblock %}
+{% block comma %}, {% endblock %}
+{% block oxford_comma %}, {% endblock %}
+
+{% block open %}Açık{% endblock %}
+{% block close %}Kapalı{% endblock %}
+
+{% macro figure_text(metadata, chapter_config, id) %}Figure {{ figure_id(metadata, chapter_config, id) }}.{% endmacro %}
+{% macro show_description(metadata, chapter_config, id) %}Şekil açıklamasını gösterin {{ figure_id(metadata, chapter_config, id) }}{% endmacro %}
+{% macro hide_description(metadata, chapter_config, id) %}Şekil açıklamasını gizleyin {{ figure_id(metadata, chapter_config, id) }}{% endmacro %}
+
+{% block written_by_before %}Yazan{% endblock %}
+{% block reviewed_by_before %}Gözden geçiren{% endblock %}
+{% block translated_by_before %}Çeviren{% endblock %}
+{% block analysis_by_before %}Analiz eden{% endblock %}
+{% block edited_by_before %}Düzenleyen{% endblock %}
+{% block reviewed_and_analyzed_by_before %}Gözden geçiren ve analiz eden{% endblock %}
+{% block analysis_and_editing_by_before %}Analiz eden ve düzenleyen{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+{% block analysis_by_after %}{% endblock %}
+{% block edited_by_after %}{% endblock %}
+{% block reviewed_and_analyzed_by_after %}{% endblock %}
+{% block analysis_and_editing_by_after %}{% endblock %}
+
+{% block unedited %}[Düzenlenmiş olan]{% endblock %}
+
+{% block author %}Yazar{% endblock %}
+{% block authors %}Yazarlar{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}}Twitter üzerinde{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}}GitHub üzerinde {% endmacro %}
+{% macro onLinkedIn(authorName) %}{{authorName}} LinkedIn üzerinde {% endmacro %}
+{% macro website(authorName) %}{{authorName}} website{% endmacro %}
+
+{% macro edition(year) %}{{ year }} Baskısı{% endmacro %}
+
+{% block footer_title %}Footer navigasyon{% endblock %}
+{% block copyright %}© Web Almanac. <a hreflang="en" href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0 </a>. altında lisanslanmıştır.{% endblock %}
```

Comment:
This this right? It creates a full stop in the middle of this, and then the next sentence starts with a lowercase a:

![Turkish copyright notice](https://user-images.githubusercontent.com/10931297/113272259-5793f080-92d3-11eb-8dc9-67eb88083e45.png)

Reply:
I have to remove the second period before the lowercase a.